src/UI/TextField.elm

```diff
@@ -492,6 +495,7 @@ renderElement cfg (TextField prop opt) =
                 ContentSpellChecked ->
                     inputAnyOptions cfg msg prop opt
                         |> Input.spellChecked elAttrs
+                        |> textFieldError cfg opt.errorCaption
```

Comment:
since you're adding it to every case doesn't it make sense to add it before the pattern match or where `nonStatic` is called?